sanic/worker/manager.py

```diff
@@ -183,6 +183,11 @@ def kill(self):
         raise ServerKilled
 
     def shutdown_signal(self, signal, frame):
+        if self.terminated:
+            logger.info("Shutdown interrupted. Killing.")
+            with suppress(ServerKilled):
+                self.kill()
+
```

Comment:
Do we want a final `return` at the end of this if statement?

Reply:
Not needed. `self.kill` ends with a `raise`